lib/containers/k8s.pm

```diff
@@ -143,7 +150,9 @@ Wait until the job is complete
 
 sub find_pods {
     my ($query) = @_;
-    return script_output("kubectl get pods --no-headers -l $query -o custom-columns=':metadata.name'");
+    my $pod = script_output("kubectl get pods --no-headers -l $query -o custom-columns=':metadata.name'");
+    record_info('Pod', "Container (POD) successfully created.\n$pod");
```

Comment:
A message about a successfully created pod in a subroutine called "find_pods" is IMHO misplaced. I think it's better to move this `record_info` to `run_container_in_k3s.pm` and keep the library methods clean, otherwise we get this info box in every other test run that uses `find_pods`.

Reply:
done